test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -127,6 +128,22 @@ class NavigationBlockUtils {
 		} );
 	}
 
+	/**
+	 * Get latest navigation menus
+	 *
+	 * @return {string} Menu content.
+	 */
+	async getNavigationMenus() {
+		const navigationMenus = this.requestUtils.rest( {
```

Comment:
Let's await it in case we want to add anything in between.

```suggestion
		const navigationMenus = await this.requestUtils.rest( {
```